frontend/stories/inputs/select/FormSearchSingleSelect.stories.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { FormSingleSelect } from '@controls/formSearchSelect/formSingleSelect/formSingleSelect.component';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { useForm } from 'react-hook-form';
+import ClearIcon from '@assets/icons/controls/clear_circle.svg';
+import ChevronIcon from '@assets/icons/chevron.svg';
+import CalendarIcon from '@assets/icons/outlined/calendar-outlined.svg';
+import PrintIcon from '@assets/icons/print.svg';
+import { SingleSelectMenuItem } from '@controls/formSearchSelect/formSingleSelect/singleSelectMenutItem.styles';
+import { FormContainer } from '../FormInput.styles';
+import { IconContainer, ItemContainer } from './FormSearchSelect.styles';
+
+export default {
+	title: 'Inputs/Select/FormSingleSelect',
```

Comment:
I decided to take the "search" out of the name because that functionality is optional and can be disabled by passing the prop `search` set to `false`
![image](https://user-images.githubusercontent.com/43547703/195852075-f39baa28-c2f8-4f7b-9dc7-59eac9d05a0c.png)

Do you reckon the `FormSingleSelect` and the `FormMultiSelect` should still include it in their name?